### DISCUSSION TITLE
GTD Norg workspace annoying error

### DISCUSSION THREAD

**Discussion by MyraBaba** [nvim-neorg/neorg#460] [Q&A] [answered]
Hi 

norg gtd always complaining Workspace not defined. 

couldnt find a solution

Best
_Lunarvim rolling and neovim 0..8_

Here my config 👍 
``
lvim.plugins = {
  { "folke/tokyonight.nvim" },
  { "lunarvim/colorschemes" },
  { "arcticicestudio/nord-vim" },
  {
    "folke/trouble.nvim",
    cmd = "TroubleToggle",
  },
  {
    "folke/todo-comments.nvim",
    event = "BufRead",
    config = function()
      require("todo-comments").setup()
    end,
  },

  {
    "nvim-neorg/neorg",
    tag = "*",
    config = function()
      require('neorg').setup {
        load = {
          ["core.defaults"] = {},
          ["core.norg.concealer"] = {}, -- Allows for use of icons
          ["core.gtd.base"] = {
            configs = {
              -- Note that this table is optional and doesn't need to be provided
            }
          },
          ["core.gtd.ui"] = {},
          ["core.keybinds"] = { -- Configure core.keybinds
            config = {
              default_keybinds = true, -- Generate the default keybinds
              neorg_leader = "<Leader>o"
            }
          },

          ["core.gtd.helpers"] = {},
          ["core.gtd.queries"] = {},
          ["core.norg.qol.toc"] = {
            config = { -- Note that this table is optional and doesn't need to be provided
              -- Configuration here
            }
          },
          ["core.norg.completion"] = {
            config = { -- Note that this table is optional and doesn't need to be provided
              engine = "nvim-cmp"

            }
          },
          ["core.highlights"] = {
            config = { -- Note that this table is optional and doesn't need to be provided
              -- Configuration here
            }
          },
          ["core.norg.dirman"] = {
            config = {
              workspaces = {
                work = "~/notes/work",
                home = "~/notes/home",
              },
              autochdir = true,

            }
          }
        }

      }
    end,
    requires = "nvim-lua/plenary.nvim"

  },
  {
    "itchyny/vim-cursorword",
    event = { "BufEnter", "BufNewFile" },
    config = function()
      vim.api.nvim_command("augroup user_plugin_cursorword")
      vim.api.nvim_command("autocmd!")
      vim.api.nvim_command("autocmd FileType NvimTree,lspsagafinder,dashboard,vista let b:cursorword = 0")
      vim.api.nvim_command("autocmd WinEnter * if &diff || &pvw | let b:cursorword = 0 | endif")
      vim.api.nvim_command("autocmd InsertEnter * let b:cursorword = 0")
      vim.api.nvim_command("autocmd InsertLeave * let b:cursorword = 1")
      vim.api.nvim_command("augroup END")
    end
  },
}

``

**Comment by max397574** [comment]
you need to set a workspace in your gtd config

**Comment by MyraBaba** [comment, marked as answer]
I tried workspace = ”my_workspace”

But same error

**Reply by mrossinek** [reply]
The name that you set as `workspace` needs to match one of the workspaces which you have actually defined in your `core.norg.dirman` configuration. In your specific case it looks like you have two workspaces: `home` and `work`.
So either of those should work. E.g.:
```lua
        ["core.gtd.base"] = {
            config = {
                workspace = "work",
            }
        },

```

**Comment by MyraBaba** [comment]
@mrossinek  Cant we have more than one workspace ?

**Reply by mrossinek** [reply]
As far as I know, the gtd module does not support that. But I do not use it in the current state myself, so @danymat is the better person to answer this question :+1:

**Comment by MyraBaba** [comment]
ok found the spell error thanks
